relentless/simulate/dilute.py

```diff
@@ -135,9 +164,24 @@ class Dilute(simulate.Simulation):
     physics of the dilute simulation.
 
     """
+
+    def _new_instance(self, initializer, potentials, directory):
+        sim = super()._new_instance(initializer, potentials, directory)
+
+        # initialize
+        initializer(sim)
+        ens = sim[initializer].ensemble
+        if isinstance(ens.V, extent.Volume):
+            sim.dimension = 3
+        elif isinstance(ens.V, extent.Area):
+            sim.dimension = 2
+        sim.types = ens.types
+
+        return sim
+
     # initialization
-    InitializeFromFile = NullOperation
-    InitializeRandomly = NullOperation
+    # InitializeFromFile = simulate.NotImplementedOperation
```

Comment:
Should this be commented out?

Reply:
I commented it out to make it clear that this was intentional not to support InitializeFromFile as an operation. We could uncomment it to be explicit, but it is slightly redundant because it is already NotImplemented in the base class. Which do you prefer?

Reply:
That makes sense. I think this is fine since it still has NotImplemented in the base class!